src/core.js

```diff
@@ -1528,6 +1528,8 @@ define([
 
         this.showContentRight();
         $props.addClass("hide");
+        var questionName = mug.getDisplayName(this.data.core.currentItextDisplayLanguage);
```

Comment:
I lean towards id only because it's simpler, but I don't feel strongly about it.

Updated to change on label change...I'd been lazy about that, but yeah, it's a better experience.